src/app/app.module.ts

```diff
@@ -18,29 +18,32 @@ import { PasswordResetComponent } from './pages/password-reset/password-reset.co
 import { EnterResetPasswordComponent } from './pages/enter-reset-password/enter-reset-password.component';
 import { AuthGuard } from './guards/auth.guard';
 import { NoAuthGuard } from './guards/no-auth.guard';
-
+import { InternationalPaymentStatusComponent } from './pages/payment/international-payment-status/international-payment-status.component';
```

Comment:
Hey @patrickf949, we should use an absolute import here

Reply:
Thanks @CryceTruly done